src/networks/blast.ts

```diff
@@ -0,0 +1,78 @@
+import { getProviderForNetwork, proxyRpcEndpoint } from '@/handlers/web3';
+import { Network, NetworkProperties } from './types';
+import { gasUtils } from '@/utils';
+import { blast } from 'viem/chains';
+import { getBlastGasPrices } from '@/redux/gas';
+import { getRemoteConfig } from '@/model/remoteConfig';
+import { BLAST_MAINNET_RPC } from 'react-native-dotenv';
+import { BLAST_ETH_ADDRESS } from '@/references';
+
+const BLAST_CHAIN_ID = 81457;
+
+export const getBlastNetworkObject = (): NetworkProperties => {
+  const { blast_enabled, blast_tx_enabled } = getRemoteConfig();
+  return {
+    // where wagmi chain data usually is
+    ...blast,
+
+    // network related data
+    enabled: blast_enabled,
+    name: 'Blast',
+    network: 'blast',
+    longName: 'Blast',
+    value: Network.blast,
+    networkType: 'layer2',
+    blockTimeInMs: 5_000,
+
+    nativeCurrency: {
+      ...blast.nativeCurrency,
+      address: BLAST_ETH_ADDRESS,
+    },
+
+    balanceCheckerAddress: '',
+    rpc: proxyRpcEndpoint(BLAST_CHAIN_ID),
+    getProvider: getProviderForNetwork(Network.blast),
+
+    // features
+    features: {
+      txHistory: true,
+      flashbots: false,
+      walletconnect: true,
+      swaps: false,
+      nfts: true,
+      pools: false,
+      txs: blast_tx_enabled,
+    },
+
+    gas: {
```

Comment:
supposedly blast is an optimism chain? we need to add the flag here so we add in the xtra gas prices

Reply:
actually jk idk where i heard this